forge-game/src/main/java/forge/game/GameAction.java

```diff
@@ -1585,6 +1590,18 @@ private boolean stateBasedAction704_attach(Card c, CardCollection unAttachList)
         return checkAgain;
     }
 
+    private boolean stateBasedAction_Contraption(Card c) {
+        //There isn't a rule-based source for this SBA, but it seems like something that should exist given the one for Space Sculptor.
```

Comment:
They said this on the Unstable mechanics article:
> In some very unusual cases (and in this set, that's saying something), a Contraption will find its way onto the battlefield without being assembled. If that ever happens, the card either goes to the scrapyard if it's an actual Contraption card or it goes to the graveyard if it's not (for example, a creature card that became a copy of a Contraption).

Reply:
Ahhhh, I'd been reading the wiki article which doesn't mention that bit. Good catch.